components/camel-sql/src/main/docs/sql-component.adoc

```diff
@@ -357,7 +357,7 @@ from("direct:query")
     .to("mock:query");
 ----
 
-If the dynamic list of values is stored in the message body, you can use `(:#in:$+{body}+)` to
```

Comment:
If I understand the AsciiDoc (https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#escaping-substitutions) docs correctly, it should be:

```
`(:#in:$++{body}++)`
```

No?